js/src/lib/inferenceSnippets/inputs.ts

```diff
@@ -81,16 +83,24 @@ const modelInputSnippets: {
 	"token-classification":     inputsTokenClassification,
 	"translation":              inputsTranslation,
 	"zero-shot-classification": inputsZeroShotClassification,
+	"image-classification":     inputsImageClassification,
```

Comment:
Can we keep the list alphabetically ordered 🥺 ?

Reply:
handled in https://github.com/huggingface/hub-docs/pull/106/commits/d4065d414c6f121fa6ff297537c7b2d77ff7ca34